entities/cerbos/policies/bulk_download.yaml

```diff
@@ -8,17 +6,12 @@ resourcePolicy:
     - common_roles
   resource: "bulk_download"
   rules:
-    - actions: ['view']
-      effect: EFFECT_ALLOW
-      derivedRoles:
-        - project_viewer
-
-    - actions: ['view', 'create', 'update']
+    - actions: ['create', 'update']
```

Comment:
Will this still work if we don't give *any* privileges to `project_member` ? All the privileges on these objects belong with the user who creates them and none with the project.

Reply:
I think that works, so do we essentially want:
```yaml
    - actions: ['*']
      effect: EFFECT_ALLOW
      derivedRoles:
        - owner
```
and nothing for `project_member`?

Reply:
Actually, if we still have bulkDownload attached to a public-ish project like Ryan suggested, we would at least need
```yaml
    - actions: ['create']
      effect: EFFECT_ALLOW
      derivedRoles:
        - project_member
```
so the user can actually create a bulkDownload in that project.

I think if we went the alternative of making `collectionId` nullable, we could probably remove it.

Reply:
OK, I think we got there 😅 

* `collectionId` is now nullable
*  Added a new `bulk_download_owner` [derivedRole](https://github.com/chanzuckerberg/czid-platformics/pull/279/files#diff-a763e89b757a4b70343a1adb3958d253e908cc7afd74c8886fc019026cc932afR38) that checks that the `collection_id` in the request is null
* [All actions on bulkDownload require this role](https://github.com/chanzuckerberg/czid-platformics/pull/279/files#diff-518d0eecf57bd9f1d6582d8acd8b704d037b2cd41a9872dada833d656dabf5efR11) now

Huge thanks to @rzlim08 for helping me with the cerbos config 🎉